### PR TITLE
projects: Add coala web project

### PR DESCRIPTION
**Pull request by hemangsk**

### CODE REVIEW

File: data/projects.js [hunk 1]
```diff
@@ -126,5 +126,26 @@ coala should support generating metrics for your code.",
...
 		"markdown": "nested_languages.md"
+	},
+	{
+		"name" : "Improving coala website & supporting tools",
+		"desc" : "This project aims to intensify usage of coala by developing \
+		well organised web interfaces.",
+		"requirements" : [
+			"At least one patch to the coala core should be accepted and merged.",
+			"The student should be familiar with ongoing technical cEPs.",
+			"The proposal needs to contain a clear concept."
```
Comment (sils) [inline data/projects.js]: the requirements have nothing to do with the project, makes no sense

File: data/projects/coala_web.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+This project aims to intensify usage of coala by developing well organised
+interfaces on web.
+
+Usually when first time users land on coala website, they want to know how to install it, run it, get the results as soon as possible. It'd be helpful if the website offers a bear search mechanism. Users can search bears and filter the search results on basis of language, its ability to detect and fix code problems and more such parameters. Furthermore the website can show analytics related to Bears, like the most popular bears, recently created bears and more.The contributors and community are an essential part of coala. The website can show coala activity of contributors in an interactive way, like having a dashboard type view showing data of each contributor.
```
Comment (sils) [inline data/projects/coala_web.md]: use line breaks, 80 chars per line max please

File: data/projects.js (hunk 1 above)
Comment (sims1253) [inline data/projects.js]: probably something like some web dev stuff but I don't feel like this is a super complicated thing that needs deep requirements

File: data/projects/coala_web.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+Lots of amazing things happen in coala at the same time. This project also aims to develop the coala blog and push it towards becoming a platform to share these stories. A better designed platform in terms of user interface and experience.
+
+We currently have coala-html to view coala analysis results in the browser in a beautiful format, out of CLI world. But some of its components require user interaction with CLI, for eg: starting up coala-html requires ```coala-html -h```. If coala-html is converted to a cross platform desktop application using frameworks such as Electron, we can have a CLI independent tool.
```
Comment (sims1253) [inline data/projects/coala_web.md]: wasn't this changed together with the jscon and ci options?

**Comment by hemangsk** [comment]
Updating requirements

File: data/projects/coala_web.md (hunk 3 above)
Comment (sils) [inline data/projects/coala_web.md]: no it wasn't, coala-html is even an own package

File: data/projects.js [hunk 4]
```diff
@@ -126,5 +126,26 @@ coala should support generating metrics for your code.",
 		"initiatives" : ["GSoC"],
 		"tags" : ["Core", "Bears"],
 		"markdown": "nested_languages.md"
+	},
+	{
+		"name" : "Improving coala website & supporting tools",
```
Comment (sils) [inline data/projects.js]: Improve -> imperative

File: data/projects.js [hunk 5]
```diff
@@ -126,5 +126,26 @@ coala should support generating metrics for your code.",
...
+		"difficulty" : "medium",
+		"issues" : [
+			"https://gitlab.com/coala/landing/issues/3"
+		],
+		"mentors" : [
+			"sils",
+			"AbdealiJK"
+		],
+		"initiatives" : ["GSoC"],
+		"tags" : ["Web-Presence", "Blog"],
```
Comment (sils) [inline data/projects.js]: Web-Presence? Maybe just Web?

File: data/projects/coala_web.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
+This project aims to intensify usage of coala by developing well organised \
```
Comment (sils) [inline data/projects/coala_web.md]: no ned for backslashes in md files!

File: data/projects/coala_web.md [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+interfaces on web.
+
+Usually when first time users land on coala website, they want to know how \
+toinstall it, run it, get the results as soon as possible. It'd be helpful if \
```
Comment (sils) [inline data/projects/coala_web.md]: missing space, please proofread your text!

File: data/projects/coala_web.md [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+
+Lots of amazing things happen in coala at the same time. This project also \
+aims to develop the coala blog and push it towards becoming a platform to \
+share these stories. A better designed platform in terms of user interface \
+and experience.
+
+We currently have coala-html to view coala analysis results in the browser \
+in a beautiful format, out of CLI world. But some of its components require \
+user interaction with CLI, for eg: starting up coala-html requires \
+```coala-html -h```. If coala-html is converted to a cross platform \
```
Comment (sils) [inline data/projects/coala_web.md]: use only one backtick on each side this is not a full code block

File: data/projects/coala_web.md [hunk 9]
```diff
@@ -0,0 +1,48 @@
...
+
+* The website is in a stable state with proper commits following coala \
+guidelines.
+* The blog is has sections for each teams in coala and teams are able \
+to post.
+* Both of them are properly tested and documented.
+
+##### GSOC 2017 FINAL
+
+* coala-html can be used a standalone desktop application.
```
Comment (sils) [inline data/projects/coala_web.md]: backticks around coala-html

**Comment by sils** [comment]
ack https://github.com/coala/projects/pull/80/commits/bcd2c653d5b05f23cc8deff7415b5e6d23fd7c3f

**Pull request merged**